ServerCore/Pages/Pieces/InternMeta.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@page "/{eventId}/{eventRole}/Teams/{teamId}/InternMeta/{puzzleId}"
+@model ServerCore.Pages.Pieces.InternMetaModel
+
+@{
+    ViewData["Title"] = "Intern Puzzleday Meta";
+}
+
+<h2>Metapuzzle Pieces Earned</h2>
+
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.EarnedPieces[0].Contents)
```

Comment:
Should this be a more meaningful label to players?

Reply:
Open to suggestions

Reply:
I'll just remove it, there's no need to keep it for a single column table.